vim-tmux-navigator.tmux

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+bind_keys() {
+  local is_vim="ps -o state= -o comm= -t '#{pane_tty}' \
+      | grep -iqE '^[^TXZ ]+ +(\\S+\\/)?g?(view|n?vim?x?)(diff)?$'"
+  tmux bind-key -n C-h if-shell "$is_vim" "send-keys C-h"  "select-pane -L"
+  tmux bind-key -n C-j if-shell "$is_vim" "send-keys C-j"  "select-pane -D"
+  tmux bind-key -n C-k if-shell "$is_vim" "send-keys C-k"  "select-pane -U"
+  tmux bind-key -n C-l if-shell "$is_vim" "send-keys C-l"  "select-pane -R"
+  tmux bind-key -n C-\ if-shell "$is_vim" "send-keys C-\\" "select-pane -l"
+}
+
+bind_keys
```

Comment:
Yes, it is. It's for not dirtying the global scope and a little bit of abstraction.
However, it would be more straightforward and simpler without the function definition.
I'm not sure which is better.